### PR TITLE
Add file for delete scm deleted branch and for pipeline source changes

### PR DESCRIPTION
**Pull request by nitind123**
Add file for delete scm deleted branch and for pipeline source changes

### CODE REVIEW

File: tests/yaml/values/config/PS-Sync-9494.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+resources:
```
Comment (vishnurao12) [inline tests/yaml/values/config/PS-Sync-9494.yml:1]: why is this file here?

File: tests/yaml/WH_DeleteBranch_8878.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+template: true
+valuesFilePath: ../values/gitRepoValues.yml
+
+resources:
+  - name: myPropertyBag
```
Comment (vishnurao12) [inline tests/yaml/WH_DeleteBranch_8878.yml:5]: names need to be unique

Comment (vishnurao12) [reply tests/yaml/WH_DeleteBranch_8878.yml:5]: for all resource, pipelines, steps

**Review by vishnurao12 — COMMENTED** [review]
I'm not sure what is the test case here. So, I have reviewed only the syntax part of it.

File: tests/yaml/values/config/PS-Sync-9494.yml (hunk 1 above)
Comment (nitind123) [reply tests/yaml/values/config/PS-Sync-9494.yml:1]: Need to add file in yaml ... as per Pipeline source API changes which read file from SCM folder.(PIPE-9494)

**Review by vishnurao12 — APPROVED** [review]
